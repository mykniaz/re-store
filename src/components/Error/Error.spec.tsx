import * as React from 'react';
import Error from './Error';
import reactTestRenderer from 'react-test-renderer';

test('Errors render test', () => {
  const component = reactTestRenderer.create(<Error />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
