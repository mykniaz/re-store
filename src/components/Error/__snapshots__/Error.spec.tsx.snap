// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors render test 1`] = `
<div>
  Error!
</div>
`;
